modules/oadp-restic-issues.adoc

```diff
@@ -99,13 +99,43 @@ $ oc get all -n <namespace>
 [id="restic-backup-cannot-be-recreated-after-s3-bucket-emptied_{context}"]
 == Restic Backup CR cannot be recreated after bucket is emptied
 
-If you create a Restic `Backup` CR for a namespace, empty the S3 bucket, and then recreate the `Backup` CR for the same namespace, the recreated `Backup` CR fails.
+If you create a Restic `Backup` CR for a namespace, empty the object storage/bucket, and then re-create the `Backup` CR for the same namespace, the re-created `Backup` CR fails.
 
-The `velero` pod log displays the error message, `msg="Error checking repository for stale locks"`.
+The `velero` pod log displays the following error message: `stderr=Fatal: unable to open config file: Stat: The specified key does not exist.\nIs there a repository at the following location?`
 
 .Cause
 
-Velero does not create the Restic repository from the `ResticRepository` manifest if the Restic directories are deleted on object storage. See (link:https://github.com/vmware-tanzu/velero/issues/4421[Velero issue 4421]) for details.
+Velero does not re-create or update the Restic repository from the `ResticRepository` manifest if the Restic directories are deleted from object storage. For more information, see link:https://github.com/vmware-tanzu/velero/issues/4421[Velero issue 4421].
```

Comment:
Very small nit but useful to know. OCP generally uses this wording: See (link) for more information/details/whatever. 

It puts the most important part of the sentence at the beginning. It's a small change but it improves usability.